db/migrate/20160813180856_add_bot_fields_to_conversation.rb

```diff
@@ -0,0 +1,14 @@
+class AddBotFieldsToConversation < ActiveRecord::Migration[5.0]
+  def change
+    add_column :conversations, :special_requests, :text
+    add_column :conversations, :additional_passengers, :integer
```

Comment:
No I need it to be null so I know whether or not we've asked about it and
gotten a response.

On Mon, Aug 15, 2016 at 11:16 AM, John McGrath <notifications@github.com>
wrote:

> In db/migrate/20160813180856_add_bot_fields_to_conversation.rb
> <https://github.com/DevProgress/drive.vote/pull/64#discussion_r74806669>:
>
> > @@ -0,0 +1,14 @@
> > +class AddBotFieldsToConversation < ActiveRecord::Migration[5.0]
> > +  def change
> > +    add_column :conversations, :special_requests, :text
> > +    add_column :conversations, :additional_passengers, :integer
>
> default 0?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/DevProgress/drive.vote/pull/64/files/ddc41bfeefca580c93c6b12be6080d6e824f82ae#r74806669>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ATwgOMZNk6DU5jPoGlYBqvDG8zTeGbBYks5qgKzmgaJpZM4Jj2K3>
> .
>